### PR TITLE
Add Styles with @emotion

### PR DESCRIPTION
**Pull request by lmachens**
In preparation for all components and UI.

### CODE REVIEW

**Review by manuelsanchez2 — COMMENTED** [review]
gj!

**Comment by EmilyCook321** [comment]
lgtm!

File: client/.storybook/preview.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
 };
+
+const withGlobalStyles = (Story, context) => {
+  return (
+    <>
+      <GlobalStyles />
+      <Story {...context} />{" "}
```
Comment (mpagels) [inline client/.storybook/preview.js]:
```suggestion
      <Story {...context} />
```
;-)

**Review by mpagels — APPROVED** [review]
gj

File: client/.storybook/preview.js (hunk 1 above)
Comment (lmachens) [reply client/.storybook/preview.js]: good catch

**Pull request merged**